test/table.test.js

```diff
@@ -1,35 +1,135 @@
-const createTestTable = require('./table');
```

Comment:
Using `"`over `'` is probably personal preference; but sure let's go with `"` from now on.